docs/plugins.rst

```diff
@@ -0,0 +1,11 @@
+Gordon's Plugin System
+======================
+
+.. automodule:: gordon.plugins_loader
+
+
+Writing a Plugin
+----------------
+.. todo::
+
+    Add documentation once interfaces are firmed up
```

Comment:
This renders like so:

![2017-12-28 at 4 56 pm](https://user-images.githubusercontent.com/1191069/34424228-0ce72bf2-ebf0-11e7-96b2-1ef775db7309.png)